jans-cedarling/cedarling/src/jwt/decoding_strategy.rs

```diff
@@ -106,7 +100,7 @@ impl DecodingStrategy {
     ///
     /// # Errors
     /// Returns an error if the claims cannot be extracted.
-    pub fn extract_claims<T: DeserializeOwned>(jwt_str: &str) -> Result<T, TokenValidationError> {
+    pub fn extract_claims<T: DeserializeOwned>(jwt_str: &str) -> Result<T, Error> {
```

Comment:
Why you remove using `TokenValidationError` ?

Reply:
I didn't really remove it, it's the `decoding_strategy::Error` now

Reply:
now `decoding_strategy::DecodingError` after [b431062](https://github.com/JanssenProject/jans/pull/9999/commits/b431062461dd38e3ba1abb4d2f650b89432ccfe1)